src/FNA3D_Driver_Vulkan.c

```diff
@@ -161,6 +161,8 @@ static inline void CreateDeviceExtensionArray(
 
 #define DEFAULT_PIPELINE_CACHE_FILE_NAME "FNA3D_Vulkan_PipelineCache.blob"
 
+#define WINDOW_SWAPCHAIN_DATA "swapchainData"
```

Comment:
Might need something more obvious for anyone inspecting via debugger:

```suggestion
#define WINDOW_SWAPCHAIN_DATA "FNA3D_VulkanSwapchain"
```

This will help differentiate it from, say, a D3D11 swapchain pair whenever that gets supported.